project-base/app/tests/FrontendApiBundle/Functional/Order/graphql/CreateFullOrderMutation.graphql

```diff
@@ -43,7 +44,8 @@ mutation CreateFullOrder(
             deliveryStreet: $deliveryStreet
             deliveryCity: $deliveryCity
             deliveryCountry: $deliveryCountry
-            deliveryPostcode: $deliveryPostcode
+            deliveryPostcode: $deliveryPostcode,
```

Comment:
we should make a final decision about commas in graphql queries... here is it really weird to have all fields without comma and one with 😄

Reply:
yeah, I have fixed it - [33f3ecc](https://github.com/shopsys/shopsys/pull/3098/commits/33f3eccde1e05aec2f07355c5e6262680a9c8fde)

Regarding the decision, I believe we need a check for that, I will create an issue :upside_down_face: